### DISCUSSION TITLE
How to view the report

### DISCUSSION THREAD

**Discussion by bollescholle** [netrunn3r/pytbull-ng#32] [Q&A]
Hi,

I used the old version of Pytbull and there was a report at the end of the test. Are these also available in Pytbull-ng and how can I see them? 

Kind regards

**Comment by juanmay1** [comment]
hi Bollescholle how are you? could you test the pytbull-ng? i can't test it, when i run the docker images (attacker and victim) they can't communicate eachother, i have followed all the steps in https://hub.docker.com/r/efigo/pytbull-ng but it doesn't work.

**Reply by netrunn3r** [reply]
Let's debug this issue here: https://github.com/netrunn3r/pytbull-ng/issues/33

**Comment by netrunn3r** [comment]
Hi Bollescholle, sorry for late response. Pytbull-ng has different approach to analyze IPS - pytbull was focused on snort/suricata from which pytbull pull reports about detected attacks and then pytbull generate his own report. My vision is that Pytbull-ng can test any IPS, so it can't pull reports from unknown IPS (or more general: UTM/NGFW), because it is impossible to write parser to recognize all IPS report formats (and I don't have access to many UTM/NGFW in the market).

My approach is:
1. create rule on UTM/NGFW which we test, to _**block**_ all attacks
2. create dedicated snort/suricata rules to detect attack performed by pytbull-ng
3. start local snort/suricata on victim
4. perform attacks from attacker to victim
5. local snort/suricata on victim check which attacks reach victim - this is first part of data to report
6. after attacks perform a survey about detected attacks by UTM/NGFW - some attacks cannot be easily blocked (eg. network anomaly) or are blocked after some time (eg. brute force), so we need info from those devices - this is second (and last) part of data to report

As we have some discussion here, whats your opinion about that? It is still in development, so to check how well IPS/IDS is working, you have to check logs on them and compare with performed attacks.

I create something like a roadmap and making reports is here: https://github.com/netrunn3r/pytbull-ng/issues/4
